### PR TITLE
Maybe fixes looms - Looming Threads now can work with 4

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Adds a = sign so that 4 cloth/cotton will work rather then needing 5+

## Why It's Good For The Game

Using 4s should work 

## Changelog
:cl:
add: Makes Looms now work with 4 sheets/cotton rather then needing 5+
add: Feedback when you can not loom an item
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
dosnt even work...

**Comment by Trilbyspaceclone** [comment]
OH

File: code/game/objects/structures/loom.dm [hunk 1]
```diff
@@ -23,11 +23,12 @@
 ///Handles the weaving.
 /obj/structure/loom/proc/weave(obj/item/stack/sheet/S, mob/user)
 	if(!istype(S) || !S.is_fabric)
+		user.show_message("<span class='notice'>This can not be loomed!</span>", MSG_VISUAL)
 		return FALSE
 	if(!anchored)
 		user.show_message("<span class='notice'>The loom needs to be wrenched down.</span>", MSG_VISUAL)
 		return FALSE
-	if(S.amount < FABRIC_PER_SHEET)
+	if(S.amount =< FABRIC_PER_SHEET)
```
Comment (Akrilla) [inline code/game/objects/structures/loom.dm:31]: Doesn't this break it and make it so you need 5?

**Comment by Akrilla** [comment]
I've tested this on master and I can 100% weave with 4 pieces of cloth.

File: code/game/objects/structures/loom.dm (hunk 1 above)
Comment (Akrilla) [reply code/game/objects/structures/loom.dm:31]: and it's <=

**Comment by Trilbyspaceclone** [comment]
i dont know what my issue is then

**Pull request closed without merging**